Bio/bgzf.py

```diff
@@ -476,13 +540,13 @@ def _load_bgzf_block(handle, text_mode=False):
     if expected_size != len(data):
         raise RuntimeError("Decompressed to %i, not %i" % (len(data), expected_size))
     # Should cope with a mix of Python platforms...
-    crc = zlib.crc32(data)
-    if crc < 0:
-        crc = struct.pack("<i", crc)
+    checksum = zlib.crc32(data)
+    if checksum < 0:
+        crc = struct.pack("<i", checksum)
     else:
-        crc = struct.pack("<I", crc)
+        crc = struct.pack("<I", checksum)
     if expected_crc != crc:
```

Comment:
You still have some ``crc`` usage despite changing the varible name above...

Reply:
This is deliberate. Static type checkers won't like `crc` being of two different types:
 - `crc` is first inferred to be an `int` from the line `crc = zlib.crc32(data)`
 - Type checkers will subsequently complain on the line `crc = struct.pack(...)`, because `struct.pack(...)` returns a `bytes` object.

Reply:
Ah, I've hit that kind of issue myself - using a new variable name does seem to be clearest. I may have not read this change carefully enough.